.github/workflows/ci.yml

```diff
@@ -34,7 +34,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        # todo put back?
```

Comment:
Maybe worth being a little more elaborate here 


(I'm assuming you're testing ci, just a little reminder to change before merging)

Reply:
i'm trying to workaround a bug in node.js that got triggered here: https://github.com/transloadit/uppy/commit/f639a6745ec45c0fafe040a049ec4b4f0dec4400

Reply:
meaning i'm trying to "fix" ci on main. not sure if 18.x, 20.x is the right way to go. I mean we use yarn.lock for all npm packages, so why shouldn't we lock node.js versions too?